src/Operator/Constructors.jl

```diff
@@ -712,6 +815,7 @@ Convenience constructor for advection operator
 - `basis`: basis for all operator fields to use
 - `mesh`:  mesh for operator
 - `wind`:  advection speed in 2D
+- `parameters`:  named tuple for wind speed
```

Comment:
The wind parameter on the line above should be removed
```suggestion
- `parameters`:  named tuple of model parameters, defines wind speed
```

Reply:
Also, `parameters` should be added to the args list on line 807